SecurityPkg/Tcg/Tcg2Pei/Tcg2Pei.c

```diff
@@ -307,7 +307,7 @@ SyncPcrAllocationsAndPcrMask (
   {
     DEBUG ((DEBUG_INFO, "TpmActivePcrBanks & Tpm2PcrMask = 0x%08x\n", (TpmActivePcrBanks & Tpm2PcrMask)));
     DEBUG ((DEBUG_INFO, "TpmActivePcrBanks & BiosHashAlgorithmBitmap = 0x%08x\n", (TpmActivePcrBanks & BiosHashAlgorithmBitmap)));
-    NewTpmActivePcrBanks  = TpmActivePcrBanks;
+    NewTpmActivePcrBanks  = TpmHashAlgorithmBitmap;

```

Comment:
This will break the feature of changing PCR banks via menu. Am I right? It always forces to use the single bank defined by PCD at coreboot build time.

Reply:
That is correct, but coreboot never extends more than a single bank and EDK2 was constrained to do the same to align the two parts of the firmware as a simpler option: https://github.com/Dasharo/dasharo-issues/issues/982

Reply:
Yes, and that should also change per my comment: https://github.com/Dasharo/edk2/pull/139#pullrequestreview-2464945893